misk/src/main/kotlin/misk/web/MiskWebModule.kt

```diff
@@ -117,6 +113,10 @@ class MiskWebModule : KAbstractModule() {
     multibind<NetworkInterceptor.Factory>(MiskDefault::class)
         .to<MarshallerInterceptor.Factory>()
 
+    // Optionally log request and response details
+    multibind<ApplicationInterceptor.Factory>(MiskDefault::class)
```

Comment:
Converted to an ApplicationInterceptor so that the request has already been unmarshalled